corehq/apps/users/views/web.py

```diff
@@ -66,10 +66,16 @@ def __call__(self, request, uuid, **kwargs):
                                       "request a project administrator to send you the invitation again."))
             return HttpResponseRedirect(reverse("login"))
 
+        is_invited_user = False
+        if request.user.is_authenticated:
+            if request.couch_user.username.lower() == invitation.email.lower():
+                is_invited_user = True
+
         if invitation.is_accepted:
-            messages.error(request, _("Sorry, that invitation has already been used up. "
-                                      "If you feel this is a mistake please ask the inviter for "
-                                      "another invitation."))
+            if request.user.is_authenticated and not is_invited_user:
```

Comment:
What happens if it is the invited user but they're not logged in?  I assume that means `is_authenticated` will be `False` and we won't be able to tell they are the invited user?

Reply:
If they're the invited user but aren't logged in, this error doesn't get added, so they just get redirected to login, and when they log in they'll go to the dashboard, which is probably what they wanted. They'll land on their last visited domain, which admittedly might be different than the invitation domain.

I suppose I could add a `next` param to force them to the invited domain, though I don't know what'll happen if  they're **not** the invited user. Maybe a 404?

Reply:
Previously any user that accessed an already-accepted invitation would get this error message, now it only appears for logged in users who are not the invitee.  I'm wondering if it would be better to continue displaying those messages for all unauthenticated users, otherwise the behavior would be kinda cryptic if it is actually a different user.  What do you think?

Reply:
Hmm, I definitely want to change _something_ about the current workflow for unauthenticated users, because for those who are the invited user, showing them an error is unnecessarily intimidating. An earlier suggestion was to switch to a more neutral message and display it in the blue "info" styles instead of red error styles. What do you think of that? (I got stuck on deciding what the message should say and removed it altogether.)

Reply:
That sounds reasonable too.  I was thinking about unauthenticated users who _are_ the invitee, but where the invite was previously accepted by someone else.  Though didn't we recently make that impossible?  Like, you have to be the invitee to accept it now?  If that's the case, then it seems fair to remove the error as you've done.

Reply:
Good point, I forgot about that PR. It's implemented as a domain setting, not default behavior, see https://github.com/dimagi/commcare-hq/pull/33447 - given that, does it still seem reasonable to keep this PR as-is?